documentation/cookbook/about.md

```diff
@@ -5,10 +5,15 @@ tags: [welcome]
 date: 2024-09-09
 ---
 
-The Einstein Platform Cookbook is where Salesforce shares example code and recipes for building with the Einstein Platform. This website is generated based on the contents of the open-source [einstein-platform](https://github.com/salesforce/einstein-platform) repository on GitHub.
+The [Einstein Platform Cookbook](https://opensource.salesforce.com/einstein-platform/) is where Salesforce shares example code and recipes for building with the Einstein Platform. This website is generated based on the contents of the open source [einstein-platform](https://github.com/salesforce/einstein-platform) repository on GitHub.
 
-Our first batch of recipes have arrived! These recipes include step-by-step instructions for implementing the LLM Open Connector with an API gateway, and each recipe features a different AI platform. More recipes are in the works featuring new AI platforms and system architectures!
+Our first batch of recipes have arrived! These recipes include step-by-step instructions for implementing the [LLM Open Connector](https://github.com/salesforce/einstein-platform?tab=readme-ov-file#llm-open-connector) with an API gateway, and each recipe features a different AI platform. More recipes are in the works featuring new AI platforms and system architectures!
```

Comment:
Can we link to the rendered version of the spec instead of the README?

```suggestion
Our first batch of recipes have arrived! These recipes include step-by-step instructions for implementing the [LLM Open Connector](/docs/apis/llm-open-connector) with an API gateway, and each recipe features a different AI platform. More recipes are in the works featuring new AI platforms and system architectures!
```

Reply:
Oh! Good point. Yes!